python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -268,7 +268,7 @@ def resolve_inner(self, asset_graph: AssetGraph) -> AbstractSet[AssetKey]:
             raise DagsterInvalidSubsetError(
                 f"AssetKey(s) {invalid_keys} were selected, but no AssetsDefinition objects supply "
                 "these keys. Make sure all keys are spelled correctly, and all AssetsDefinitions "
-                "are correctly added to the repository."
+                "are correctly added to the `Definitions`."
```

Comment:
I think user-facing error strings should mention both concepts for now. Otherwise might get super confusing for people who have not migrated.